.travis.yml

```diff
@@ -6,7 +6,7 @@ language: go
 go:
   - "1.11.x"
 env:
-  GOFLAGS=-mod=vendor
+  - GO111MODULE=on GOFLAGS=-mod=vendor
```

Comment:
don't these need to be one per line?

Reply:
no, one per line will do a run with each env var separately (I know weird..)